js/web/lib/onnxjs/backends/webgl/program-manager.ts

```diff
@@ -37,10 +37,7 @@ export class ProgramManager {
     this.repo.set(key, artifact);
   }
   run(buildArtifact: Artifact, runData: RunData): void {
-    const inputInfo = runData.inputTextureDatas.map((d, i) => `input${i}:[${d.shape}]`).join(', ');
-    const outputInfo = `output: [${runData.outputTextureData.shape}]`;
-
-    this.profiler.event('backend', `ProgramManager.run ${inputInfo} ; ${outputInfo}`, () => {
+    this.profiler.event('op', `ProgramManager.run ${buildArtifact.name}`, () => {
```

Comment:
shape information will be gone. is this information useful to see a profiling data for the same kernel for different shape?

Reply:
We can get shape info from other options like log-verbose, so remove it here to make the log concise. but we could add it back if needed.

Reply:
it's okay for now. I think we can provide some way to combine shape information with concise profiling data later.